src/main/tasks_tests_handler.py

```diff
@@ -0,0 +1,212 @@
+INPUT_FILE_NAME = consts.TASKS_TESTS.INPUT_FILE_NAME.value
+
+
+def __create_file(content: str, task: str, extension='py', source_file_name=SOURCE_FILE_NAME):
+    with open(TASKS_TESTS_PATH + task + '/' + source_file_name + '.' + extension, 'w') as f:
+        f.write(content)
+
+
+def __is_file(file: str, task: str):
+    return os.path.isfile(TASKS_TESTS_PATH + task + '/' + file)
+
+
+def __remove_file(file: str, task: str):
+    if __is_file(file, task):
+        os.remove(TASKS_TESTS_PATH + task + '/' + file)
+
+
+def __get_content_from_file(file: str, task: str):
+    with open(TASKS_TESTS_PATH + task + '/' + file, 'r') as f:
+        return f.read().rstrip("\n")
+
+
+def __get_in_and_out_files(list_of_files: list):
+    in_files_list = list(filter(lambda file_name: 'in' in file_name and '.txt' in file_name, list_of_files))
+    out_files_list = list(filter(lambda file_name: 'out' in file_name and '.txt' in file_name, list_of_files))
+    if len(out_files_list) != len(in_files_list):
+        raise ValueError('Length of out files list does not equal in files list')
+    in_and_out_pairs = __separate_in_and_out_files_on_pairs(in_files_list, out_files_list)
+    return in_and_out_pairs
+
+
+def __get_out_file_by_in_file(in_file: str):
+    return 'out_' + str(in_file.split('.')[0].split('_')[-1]) + '.txt'
+
+
+def __separate_in_and_out_files_on_pairs(in_files: list, out_files: list):
+    pairs = []
+    for in_file in in_files:
+        out_file = __get_out_file_by_in_file(in_file)
+        if out_file not in out_files:
+            raise ValueError('List of out files does not contain a file for ' + in_file)
+        pairs.append((in_file, out_file))
+    return pairs
+
+
+def __is_python(extension: str):
+    return extension == 'py'
+
+
+def __get_java_class(source_code: str):
+    class_key_word = 'class'
+    rows = source_code.split('\n')
+    for row in rows:
+        if class_key_word in row:
+            class_index = row.index(class_key_word)
+            return row[class_index + len(class_key_word) + 1:].replace(' ', '').replace('{', '')
+    raise ValueError('Source code does not contain class name!')
+
+
+# Wrap all values from input in the print command
+def __create_py_input_file(txt_in_file: str, task: str, extension='py', file_name=INPUT_FILE_NAME):
+    code = ''
+    with open(TASKS_TESTS_PATH + task + '/' + txt_in_file, 'r') as f:
+        for line in f:
+            code += 'print(' + line.strip('\n') + ')' + '\n'
+    __create_file(code, task, extension, file_name)
+
+
+# For python scripts it is an in file with extension py and for other cases, it is an in file with extension txt
+def __get_in_file_for_current_test(cur_in_file: str, task: str, extension='py'):
+    if __is_python(extension):
+        __create_py_input_file(cur_in_file, task)
+        return INPUT_FILE_NAME + '.' + extension
+    return cur_in_file
+
+
+# Clear not .txt files: we need to clear the generated files (it has not .txt extension)
```

Comment:
I'm not sure about using the world 'clear' like this because 'clear smth' means 'keep this smth, but remove the other stuff'. So talking about old files, we don't want to clear them, we want to clear the other files from them. 
Let's change it to make it clear :)